EIPS/eip-6734.md

```diff
@@ -0,0 +1,574 @@
+
+[[R5]](#r5) testability: Since each of the non-conformance-target requirements in this documents is testable, so must be the totality of the requirements in this document. Therefore, conformance tests for all requirements can exist, be described, be built and implemented and results can be recorded as required in [[R5]](#r5).
+
+-----
+
+## Rationale
+
+The standard is extending and clarifying current custom lists such as from Arbitrum and Optimism as referenced in the [Motivation](#motivation) or the Uniswap Tokenlist Project to improve clarity, security and encourage adoption by non-Web3 native entities.
+
+-----
+
+## Security Considerations
+
+There are no additional security requirements apart from the warnings that URIs utilized in implementations of this standard might be direct to malicious resources such as websites, and that implementers should ensure that data utilized for a canonical token list is secure and correct. Since this standard is focused on a data schema and its data properties there are no additional security considerations from for example homoglyph attacks (see [CVE-2021-42574](#cve-2021-42574)).
+
+### Security Considerations: Data Privacy
+
+The standard does not set any requirements for compliance to jurisdiction legislation/regulations. It is the responsibility of the implementer to comply with applicable data privacy laws.
+
+### Security Considerations: Production Readiness 
+
+The standard does not set any requirements for the use of specific applications/tools/libraries etc. The implementer should perform due diligence when selecting specific applications/tools/libraries.
+
+### Security Considerations: Internationalization and Localization
+
+The standard encourages implementers to follow the [W3C "Strings on the Web: Language and Direction Metadata" best practices guide](#w3c-string-meta) for identifying language and base direction for strings used on the Web wherever appropriate.
+
+-----
+
+### Appendix A - Normative References
+
+The following documents are referenced in such a way that some or all of their content constitute requirements of this document.
+
+<a name="rfc2119"> **[RFC2119]** </a>\
+ S. Bradner, Key words for use in RFCs to Indicate Requirement Levels, IETF RFC 2119, March 1997.
+
+<a name="jsonld"> **[JSONLD]** </a>\
+JSON-LD 1.1, M. Sporny, D. Longley, G. Kellogg, M. Lanthaler, Pierre-Antoine Champin, N. Lindström, W3C Recommendation, July 2020 
+
+<a name="rfc3986"> **[RFC3986]** </a>\
+T. Berners-Lee, R. Fielding,  L. Masinter, Uniform Resource Identifier (URI): Generic Syntax, IETF RFC 3986, January 2005
+
+<a name="eip-155"> **[EIP-155]** </a>\
+Vitalik Buterin, "EIP-155: Simple replay attack protection," Ethereum Improvement Proposals, no. 155, October 2016.
+
+<a name="eip-3220"> **[EIP-3220]** </a>\
+Weijia Zhang, Peter Robinson, "EIP-3220: Crosschain Identifier Specification [DRAFT]," Ethereum Improvement Proposals, no. 3220, October 2020.
```

Comment:
It is sufficient to list normative EIP references in the `requires:` preamble header, and directly link to them using normal markdown syntax (eg. `[EIP-3220](./eip-3220.md)`.)

```suggestion
```

Reply:
ok. did that already but for ease of reference added them in where explicitly referenced. hope that is still acceptable.